tests/common/ixia/ixia_helpers.py

```diff
@@ -0,0 +1,59 @@
+import re 
+
+"""
+@summary: given a DUT interface, return the management IP address of its neighbor IXIA device
+@param intf: DUT interface
+@param conn_graph_facts: testbed connectivity graph
+@param ixia_dev: the mapping of hostname to IP address of IXIA devices
+@return the management IP address of its neighbor IXIA device or None if we cannot find it
+"""
+def get_neigh_ixia_mgmt_ip(intf, conn_graph_facts, ixia_dev):
```

Comment:
use fanout_graph_facts intead of conn_graph_facts. that will return all the fanouts(ixia chassis) associated with the dut as a dict which will have all the details for each of the fanouts

Reply:
I have used fanout_graph_facts to get chassis/card/port information.

Reply:
I don't see any changes in this method. If we aren't using these, please remove them